src/bun.js/api/ffi.zig

```diff
@@ -706,12 +706,12 @@ pub const FFI = struct {
                     if (!value.isString()) {
                         return globalThis.throwInvalidArgumentTypeValue("source", "array of strings", value);
                     }
-                    try compile_c.source.files.append(bun.default_allocator, try (try value.getZigString(globalThis)).toOwnedSliceZ(bun.default_allocator));
+                    try compile_c.source.files.append(bun.default_allocator, try (try value.getZigString(globalThis)).toOwnedSlice(bun.default_allocator));
```

Comment:
I’m not sure if we’re running the FFI tests in ci

Reply:
`source.files` is a `std.ArrayListUnmanaged([:0]const u8)`. this is still safe because the return type of `. toOwnedSlice` was changed to return a nul-terminated slice so having a separate `. toOwnedSliceZ` was unnecessary